src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2580,86 +2580,92 @@ namespace WaterToAirHeatPumpSimple {
 
                 // determine adjusted cooling and heating coil capacity
                 simpleWatertoAirHP.RatedCapHeatAtRatedCdts = RatedCapHeatDes * RatedHeatCapTempModFac;
```

Comment:
I believe what the code below is trying to do is to size a HP, so the coils would need to be the same size. In this case that's not necessary but an actual HP would need to be tested to make sure it works. I guess it does work since all WSHPs apparently size correctly, or do they?